benchopt/utils/tests/test_parquet.py

```diff
@@ -85,11 +82,7 @@ def test_metadata_saving():
             run(run_cmd, 'benchopt', standalone_mode=False)
 
         config = get_metadata(Path(out.result_files[0]))
-        expected_config = {
-            k: v for k, v in DEFAULT_BENCHMARK_CONFIG.items()
-            if k in ['plots', 'plot_configs']
-        }
-        assert config == expected_config
+        assert config == {'plot_configs': {}}
```

Comment:
Might need to check it matches with custom plots?

Reply:
Oh, this will actually make our life harder for generating the results website (https://github.com/benchopt/results) as we will need to keep the custom plot on the metadata for the benchmark...

Need to think about this for later.